### PR TITLE
TDD Exercise

### PR DESCRIPTION
**Pull request by CheezItMan**
An exercise where students write tests and update a method to make it pass.  Lots of edge-cases to test which makes it interesting, but doesn't require complicated loops, just a basic `.each` and some if-statements.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,36 @@
 # TDD Exercise
 
+In this exercise you will write a method and set of tests in TDD fashion which calculates a hand's BlackJack score.
+
+## About BlackJack
+
+In the card name BlackJack each card has a value.
```
Comment (tildeee) [inline README.md]: I think you might mean something like "In the card game Blackjack each card has a value."?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,36 @@
...
+-  Number cards (2-10) carry the card's numeric value.
+-  Face cards on the other hand ("Jack", "Queen", "King") have a value of 10.
+-  Aces (1) can have a value of either 1 or 11, whichever will get the hand closest to 21 without going over.
+
```
Comment (tildeee) [inline README.md]: In the above paragraph, it talks about the game Blackjack in the real-world scenario and its rules.

In the below paragraph, it switches to talking about it as a program (it's described that the hand is the array `[3, "King", 1]`. Since this section is about Blackjack's rule in the real-life scenario/business scenario, and not about implementation, it may help to just say "For example, if my hand has three cards, a 3, a King, and a 1, my Blackjack score is 14 (3 + 10 + 1). If I have a two-card hand of a 1 and a Jack, then my score is 21 (11 +10)."

File: README.md [hunk 3]
```diff
@@ -1,2 +1,36 @@
...
+1.  What would be two nominal cases?
+2.  What edge cases can you determine?  Name at least 3.
+3.  How would you **test** these cases?
+
+When you finish talk with another pair and walk through your answers.
+
+## Part 2:  Calculating a Score
+
+You will write a method called:  `blackjack_score` which take an array of card values and returns the blackjack score.  The card values can be any of the following, number values 1-9, "King", "Queen", "Jack".  If the array contains an invalid card value or the total exceeds 21, raise an `ArgumentError`.  For example `blackjack_score([1, 5, 3])` will return 19.
```
Comment (tildeee) [inline README.md]: Do we want to also tell them explicitly to raise an `ArgumentError` if the hand is greater than 5 items?

File: README.md [hunk 4]
```diff
@@ -1,2 +1,36 @@
...
+In this exercise you will write a method and set of tests in TDD fashion which calculates a hand's BlackJack score.
+
+## About BlackJack
+
+In the card name BlackJack each card has a value.
+-  Number cards (2-10) carry the card's numeric value.
+-  Face cards on the other hand ("Jack", "Queen", "King") have a value of 10.
+-  Aces (1) can have a value of either 1 or 11, whichever will get the hand closest to 21 without going over.
+
+For example if I my hand is:  `[3, "King", 1]`, my BlackJack score is 14 (3 + 10 + 1).  If I have `[1, "Jack"]` then my score is 21 (11 + 10).
```
Comment (tildeee) [inline README.md]: "For example if I my hand is:" -> "For example if my hand is:"

**Review by tildeee — CHANGES_REQUESTED** [review]
Fantastic activity; I'm glad it's simple so it helps the students focus on the test part. I think it's great!

I mostly made suggestions on grammar, but my biggest suggestion is to separate the "business understanding" of Blackjack from the "implementation understanding"

**Comment by CheezItMan** [comment]
Updated

**Review by droberts-sea — APPROVED** [review]
